.gitignore

```diff
@@ -9,3 +9,14 @@ yarn-debug.log*
 yarn-error.log*
 *.log
 .history
+
+### Sass ###
+.sass-cache/
+.scss-cache/
+*.css.map
+*.sass.map
+*.scss.map
+*.css
+
+## System ##
+.DS_Store
```

Comment:
To avoid this icon to show, we leave a empty line at each of the files.

Reply:
Read the first answer of this question and you will get the importance of EOF for versioning systems like git
https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file